core/src/test/java/org/elasticsearch/index/mapper/TokenCountFieldMapperTests.java

```diff
@@ -144,4 +146,53 @@ public void testEmptyName() throws IOException {
         );
         assertThat(e.getMessage(), containsString("name cannot be empty string"));
     }
+
+    public void testParseNullValue() throws Exception {
+        DocumentMapper mapper = createIndexWithTokenCountField();
+
```

Comment:
nit: test is very short, so maybe we don't need any strutcturing empty lines in it.